dbotconf/internal/verify.go

```diff
@@ -29,37 +29,41 @@ var (
 	errNotEnoughArg = errors.New("path argument required")
 )
 
-// configuredUpdates returns the set of Go modules and Dockerfiles dependabot
+// configuredUpdates returns the set of Go modules, Dockerfiles, Pip requirements dependabot
 // is configured to check updates for.
-func configuredUpdates(path string) (mods map[string]struct{}, docker map[string]struct{}, err error) {
+func configuredUpdates(path string) (mods map[string]struct{}, docker map[string]struct{}, pip map[string]struct{}, err error) {
```

Comment:
Can we return a new struct instead of `mods map[string]struct{}, docker map[string]struct{}, pip map[string]struct{}` ? Retruning 4 values dies not look very well.